extensions/wikia/DesignSystem/scripts/DesignSystemAuthenticationMenu.js

```diff
@@ -0,0 +1,36 @@
+$(function ($) {
+	'use strict';
+
+	var $signInButton = $('#global-navigation-anon-sign-in'),
+		$registerButton = $('#global-navigation-anon-register');
```

Comment:
we have exactly the same event handler for those elements. Can we create a selector that matches both and add event listener for both?